tests/bsim/bluetooth/audio/test_scripts/cap_broadcast_ac_14.sh

```diff
@@ -41,8 +41,12 @@ Execute_AC_14 24_1_1
 Execute_AC_14 24_2_1
 Execute_AC_14 32_1_1
 Execute_AC_14 32_2_1
-Execute_AC_14 441_1_1
-Execute_AC_14 441_2_1
+# ASSERTION FAIL [err == ((isoal_status_t) 0x00) || err == ((isoal_status_t) 0x04)]
```

Comment:
Just as a note: if this is failing only for the 5340, you could if it out in that case, and still run it for the 52.

Reply:
It is only failing for the 53 series. However, it's not a big deal to disable this particular test for the 52 series.